pdp/mapper_rca.go

```diff
@@ -34,33 +34,33 @@ func getSetOfIDs(v AttributeValueType) ([]string, error) {
 	return sortSetOfStrings(IDs), nil
 }
 
-func getRulesMap(rules []RuleType, params *MapperRCAParams) map[string]*RuleType {
+func getRulesMap(rules []*RuleType, params *MapperRCAParams) map[string]*RuleType {
 	if params.RulesMap != nil {
 		return params.RulesMap
 	}
 
 	m := make(map[string]*RuleType)
 	for _, rule := range rules {
 		tmp := rule
```

Comment:
In case of pointers we don't need `tmp` variable. You need to put pointer directly to the list.